routes/update-routes.js

```diff
@@ -1,169 +1,23 @@
 /* Routes for modifying user data */
 const passport = require("passport");
-const profileService = require("../services/name-gender-bio");
-const dateService = require("../services/date");
-const locationService = require("../services/location");
-const additionalInfoService = require("../services/interests");
-const maxmind = require("maxmind");
-const fs = require("fs");
-const buffer = fs.readFileSync("./lib/GeoLite2-City.mmdb");
-const lookup = new maxmind.Reader(buffer);
+const profileService = require("../services/profile");
 
 module.exports = (app) => {
-  //This route updates name of a user in the profile
-  // app.post(
-  //   "/api/profile/name",
-  //   passport.authenticate("jwt", { session: false }),
-  //   async (req, res) => {
-  //     try {
-  //       console.log("in try")
-  //        await profileService.addName(req.user.userId, req.body.name);
-
-  //     } catch (e) {
-  //       res.status(400).send("Could not complete the request to update name");
-  //     }
-  //   })
-
-  app.post(
-    "/api/profile/name",
-    passport.authenticate("jwt", { session: false }),
-    (req, res) => {
-      console.log(req.body.name);
-      profileService
-        .addName(req.user.userId, req.body.name)
```

Comment:
this route should be called updateName or something like this, since this is what it does, because it in in update-routes? should also be PATCH or PUT then